src/pages/StatsPage/index.tsx

```diff
@@ -159,7 +160,11 @@ export function StatsPage() {
               </StatsItemIcon>
               <StatsItemData>
                 <Typography fontWeight="normal" variant="h3">
-                  {overviewInfo.BALNMarketCap ? getFormattedNumber(overviewInfo.BALNMarketCap, 'currency0') : '-'}
+                  {overviewInfo.BALNMarketCap ? (
+                    getFormattedNumber(overviewInfo.BALNMarketCap, 'currency0')
+                  ) : (
+                    <LoaderComponent />
```

Comment:
Currently, there are two placeholder animations. LoaderComponent and Skeleton. @parrot9design 

![image](https://user-images.githubusercontent.com/50083174/148739421-54504c03-bf1f-428f-ba22-8a372f0ba3f2.png)
![image](https://user-images.githubusercontent.com/50083174/148739491-9a7f6625-f213-4f2d-bac5-dbd9e2855232.png)

What about using the same animation?

Reply:
Yea, I'm not sure about this as well. I changed the first one on the top for comparison. Let Lisa and Peter decide.

Reply:
Thanks for the comparison! Happy to use two different animations here, so you can change the `Total value locked` animation back to LoaderComponent. 😊

So we can maintain a consistent animation style going forwards, what are your thoughts on this as a general rule?

- Use LoaderComponent for single values, like the data points in the overview, collateral/loan, and governance sections on the Stats page.
- Use Skeleton for content that takes up more space/has multiple data points, like the Tokens and Exchange sections on the Stats page, and the content on the Vote page.

(@hetfly if we go by this logic, we should add the Skeleton animation to the Holdings section of the [Performance details](https://stats.balanced.network/performance-details) page, as well.)

Reply:
Agreed, code updated